pkg/pods/pod.go

```diff
@@ -100,12 +101,12 @@ func (pod *Pod) Halt(manifest Manifest) (bool, error) {
 		switch err.(type) {
 		case nil:
 			// noop
-		case hoist.DisableError:
+		case launch.DisableError:
 			// do not set success to false on a disable error
-			pod.logLaunchableWarning(launchable.Id, err, "Could not disable launchable")
+			pod.logLaunchableWarning(launchable.GetID(), err, "Could not disable launchable")
 		default:
-			// this case intentionally includes hoist.StopError
-			pod.logLaunchableError(launchable.Id, err, "Could not halt launchable")
+			// this case intentionally includes launch.StopError
+			pod.logLaunchableError(launchable.GetID(), err, "Could not halt launchable")
```

Comment:
Naming getters without the "Get" is a an explicit [language convention](https://golang.org/doc/effective_go.html#Getters), but it's less clear what to do when you really want a public field with that name too. Any strong opinions on naming a public field `ID_` and its getter `ID()`?